types/wlr_xdg_foreign_registry.c

```diff
@@ -0,0 +1,77 @@
+#include <wlr/types/wlr_xdg_foreign_registry.h>
+#include "util/signal.h"
+#include "util/uuid.h"
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+
+/**
```

Comment:
The doc-comment is already in the header file. Would prefer not to duplicate it.